src/cc2olx/tools/README.rst

```diff
@@ -0,0 +1,61 @@
+Tools
+######
+
+Video Upload Tool
+-----------------
+
+The video upload tool uploads video files to edX's video encoding pipeline.
+It also generates an output CSV that can be used as input to the video xBlock conversion tool.
+
+Input
+-----
+The tool takes as command line arguments a ``course_id``, a ``directory``, and a ``csv``.
+
+The ``course_id`` argument is the ID of the course as it appears in Studio. For example, ``course-v1:edX+111222+111222``.
```

Comment:
I'm not sure if this is the canonical place to get the course ID. Is there a better place I can refer to?